### ISSUE TITLE
Tab bar doesn't load by default in private window.

### ISSUE THREAD

**Issue by JokeDeity** [ranmaru22/firefox-vertical-tabs#117]
I would like to know how to make the bar load normally as it does in regular windows in private windows.  Currently, at least for me, I have to press Shift+F1 to get it to appear in private windows, in regular windows it's just there.

Also like to reiterate on two other users posted issues that I am also experiencing:
The bar appears in popup windows like downloads or Google sign ins.
The bar does not load for me in fullscreen mode.  I theorize because I have two monitors that meet where the bar is?

**Comment by ranmaru22** [comment]
I don't think this is possible, at least not with the level of modification we are doing here. I think you need a full extension for that, and then have to give that extension the rights to run in private windows.

I suggest putting the sidebar button into your toolbar, that way you can just click it to show the tabs immediately. It's not a perfect solution but it's quicker.

I will comment on the other issues in the respective threads.

**Comment by ranmaru22** [comment]
OK, I noticed the other issues don't have threads, lol.

> The bar appears in popup windows like downloads or Google sign ins.

This is another case of where the level of customization this mod offers isn't enough to fix it. User CSS does not have access to the APIs you would need to even check what kind of window is open.

> The bar does not load for me in fullscreen mode. I theorize because I have two monitors that meet where the bar is?

Yes, without further context, I would assume that as well. The bar should open when you push your cursor against the edge of the screen, if that edge does not exist because another frame follows, I could imagine that the slide-out animation simply doesn't trigger.

Have you tried putting the sidebar on the other side and see if that works?

**Comment by JokeDeity** [comment]
Thanks a trillion for being actually responsive and helpful, you don't know how rare that is in my experience.  So, I'm finding most issues bareable as you've explained they aren't exactly fixable.  Luckily, 

1.  Private windows is a half second solution to the slight annoyance, 
2.  I can live with it showing up in the popups, but would love a solution if one can be found in the future, 
3.  I almost never use fullscreen, so it's not a big deal for me, but I did confirm it's the dual monitors because it works fine on monitor one or on the opposite side of the browser on monitor two.  Is it possible that you could expand the area it detects your mouse in or is this beyond what the script is doing/another situation where it doesn't know what's happening (ie fullscreen or not)?

**Comment by JokeDeity** [comment]
Love your work by the way, really enjoying this!

**Comment by ranmaru22** [comment]
Thank you. :)

> Is it possible that you could expand the area it detects your mouse in or is this beyond what the script is doing/another situation where it doesn't know what's happening (ie fullscreen or not)?

There is a custom property called `--fullscreen-sidebar-width` in the userChrome.css file. You can change that back to 48px (it's set to 1px by default) to have the sidebar not collapse at all in fullscreen. You can technically also set it to any other value you like, say 10px, which would give you a 10px area to interact with, but it will look kinda strange because you'll have a half-collapsed sidebar on the side.

I've played around with opacity, and make the collapsed area 10px plus transparent, so that it won't look half-there, but that will leave you with an unusable area which looks like this:

![image](https://github.com/ranmaru22/firefox-vertical-tabs/assets/16521734/61be351b-5950-4ca0-b951-5b76fe5fa9f2)

Plus is would make customizing this behavior more complex for people who want the sidebar not collapse at all in fullscreen.

**Comment by JokeDeity** [comment]
I ended up settling on setting it to 35px (the size I have the sidebar) and if I want to completely hide it in fullscreen I can use the button.
